guides/common/modules/con_single-location-with-segregated-subnets.adoc

```diff
@@ -0,0 +1,7 @@
+[id="single-locations-with-segregated-subnets_{context}"]
```

Comment:
```suggestion
[id="single-locations-with-segregated-subnets"]
```

If we introduce new anchors, I'd appreciate it if we drop the `{context}` unless necessary. I am also cool if I can do this in a follow-up. Applies to all files.

Reply:
Done. I dropped `{context}` from all the module IDs in the new assembly.